src/actions/userActions.js

```diff
@@ -138,6 +141,7 @@ export function giantswarmLogin(email, password) {
         return userData;
       })
       .then(userData => {
+        localStorage.setItem('user', JSON.stringify(userData));
```

Comment:
would it make sense to use the new setUser here? (and in a few other cases too)

Reply:
TBH I don't know, I wasn't sure why `setUser()` declare and assign `defaultClient`and `defaultClientAuth` so I left it as I found it.